src/main/java/com/linecorp/armeria/internal/grpc/NettyReadableBuffer.java

```diff
@@ -0,0 +1,100 @@
+package com.linecorp.armeria.internal.grpc;
+
+import java.io.IOException;
+import java.io.OutputStream;
+import java.nio.ByteBuffer;
+
+import io.grpc.internal.AbstractReadableBuffer;
+import io.netty.buffer.ByteBuf;
+import io.netty.buffer.Unpooled;
+
+/**
+ * A {@link java.nio.Buffer} implementation that is backed by a Netty {@link ByteBuf}. This class
+ * does not call {@link ByteBuf#retain}, so if that is needed it should be called prior to creating
+ * this buffer.
+ *
+ * Fored from grpc-java.
+ */
+public class NettyReadableBuffer extends AbstractReadableBuffer {
+    private final ByteBuf buffer;
+    private boolean closed;
+
+    public NettyReadableBuffer(byte[] data, int offset, int length) {
+        this(Unpooled.wrappedBuffer(data, offset, length));
```

Comment:
Could you explain why this method was added rather than using `ReadableBuffers.wrap()`?